pkgs/os-specific/linux/fuse/common.nix

```diff
@@ -43,50 +44,73 @@ in stdenv.mkDerivation rec {
     then [ meson ninja pkg-config ]
     else [ autoreconfHook gettext ];
 
-  outputs = [ "out" ] ++ lib.optional isFuse3 "common";
+  # The `common` output (`fuse-common` from top-level) is for `/etc` which shares
+  # between fuse2 and fuse3. From the documentation, `fuse3`'s common files are preferred,
+  # even for `fuse2`.
+  # https://github.com/libfuse/libfuse/blob/d372d3f80b5437e652ea501d8a4a917f7001b9d7/ChangeLog.rst#libfuse-300-2016-12-08
+  #
+  # It contains,
+  # - An init script `/etc/init.d/fuse3`.
+  #   This is unused for systemd-managed system.
+  #
+  # - A udev rules `/etc/udev/rules.d/99-fuse3.rules` to configure `/dev/fuse`.
+  #   This is included in systemd-udevd default rules since very early time, thus unneeded.
+  #   https://github.com/systemd/systemd/blob/0cf2dcf15402c60498165dbd3f14536766c05051/rules.d/50-udev-default.rules.in#L94
+  #
+  # - An example commented `/etc/fuse.conf`.
+  #   In NixOS, we control and generate it in `nixos/modules/programs/fuse.nix`.
+  #   Also the example actually enables nothing.
```

Comment:
I would move that file to docs personally.

Reply:
> to docs

Do you mean `doc/builders/packages/fuse.section.md`?

Reply:
I meant the docs output.

Reply:
> I meant the docs output.

If we split the current `common` output further into `udev` and `docs` outputs. What should the top-level `fuse-common` point to? It is not used in nixpkgs now though.

Reply:
Okay now I drop the vague "common" output and only keep "udev" (though they should be obsolete in favor of systemd-udevd), and the no-op config is removed from output.